test/gui/shared/scripts/pageObjects/SyncConnection.py

```diff
@@ -29,6 +29,49 @@ class SyncConnection:
         "type": "QPushButton",
         "visible": 1,
     }
+    ADD_FOLDER_SYNC_BUTTON = {
+        "container": names.settings_stack_QStackedWidget,
+        "name": "addButton",
+        "type": "QPushButton",
+        "visible": 1,
+    }
+    NAVIGATE_BACK_BUTTON = {
+        "name": "__qt__passive_wizardbutton0",
+        "type": "QPushButton",
+        "visible": 1,
+        "window": names.add_Folder_Sync_Connection_OCC_FolderWizard,
+    }
+    ADD_FOLDER_BUTTON = {
+        "container": names.add_Folder_Sync_Connection_groupBox_QGroupBox,
+        "name": "addFolderButton",
+        "type": "QPushButton",
+        "visible": 1,
+    }
+    ADD_FOLDER_INPUT = {
+        "buddy": names.create_Remote_Folder_Enter_the_name_of_the_new_folder_to_be_created_below_QLabel,
+        "type": "QLineEdit",
+        "unnamed": 1,
+        "visible": 1,
+    }
```

Comment:
AFAIK, we should already have this selector. Also, the best place for all these should be `SyncConnectionWizard`